main.go

```diff
@@ -60,7 +61,19 @@ func main() {
 			Use:   "version",
 			Short: "Print the version number of Arduino Flasher CLI",
 			Run: func(cmd *cobra.Command, args []string) {
-				fmt.Println("Arduino Flasher CLI v" + Version)
+				feedback.Print("Arduino Flasher CLI " + Version)
+				latest, err := checkForUpdates()
+				if err != nil {
+					feedback.Warning("\n\nfailed to check for updates: " + err.Error())
+				}
+				if latest != "" {
+					msg := fmt.Sprintf("\n\n%s %s → %s\n%s",
+						color.YellowString(i18n.Tr("A new release of Arduino Flasher CLI is available:")),
+						color.CyanString(Version),
+						color.CyanString(latest),
+						color.YellowString("https://www.arduino.cc/en/software/#flasher-tool"))
+					feedback.Print(msg)
+				}
```

Comment:
here I would use the PrintResult feature, so we will support also json format

```suggestion
				latest, err := checkForUpdates(1 * time.Second)
				if err != nil {
					feedback.Warning(color.YellowString("Failed to check for updates: "+err.Error()) + "\n")
				}
				if latest != "" {
					msg := fmt.Sprintf("%s %s → %s\n%s",
						color.YellowString(i18n.Tr("A new release of Arduino Flasher CLI is available:")),
						color.CyanString(Version),
						color.CyanString(latest),
						color.YellowString("https://www.arduino.cc/en/software/#flasher-tool"))
					feedback.Warning(msg + "\n")
				}

				feedback.PrintResult(versionResult{
					Name:    "Arduino Flasher CLI",
					Version: Version,
				})

```

versionResult should be defined like
```
type versionResult struct {
	Name    string `json:"name"`
	Version string `json:"version"`
}

func (v versionResult) String() string {
	return fmt.Sprintf("%s %s", v.Name, v.Version)
}

func (v versionResult) Data() any {
	return v
}
```